### PR TITLE
Update New-CsExternalAccessPolicy.md

### PR DESCRIPTION
**Pull request by kellybos**
Need to direct admins to Teams documentation for these settings (Note in synopsis). Also EnableOutsideAccess is applicable to Skype Server only, not Teams and Skype online. Confirmed with engineering and okay to update documentation to make this more clear.

### CODE REVIEW

File: skype/skype-ps/skype/New-CsExternalAccessPolicy.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Enables you to create a new external access policy.
 External access policies determine whether or not your users can: 1) communicate with users who have Session Initiation Protocol (SIP) accounts with a federated organization; 2) communicate with users who have SIP accounts with a public instant messaging (IM) provider such as MSN; and, 3) access Skype for Business Server over the Internet, without having to log on to your internal network.
 This cmdlet was introduced in Lync Server 2010.
 
-
+For information about external access in Microsoft Teams, please see https://docs.microsoft.com/en-us/microsoftteams/manage-external-access and https://docs.microsoft.com/en-us/microsoftteams/teams-skype-interop for specific details.
```
Comment (get-itips) [inline skype/skype-ps/skype/New-CsExternalAccessPolicy.md:21]:
```suggestion
For information about external access in Microsoft Teams, see [Manage external access in Microsoft Teams](/microsoftteams/manage-external-access) and [Teams and Skype interoperability](/microsoftteams/teams-skype-interop) for specific details.
```

**Comment by tseward** [comment]
Approved once the changes suggested by @get-itips is implemented.

**Review by tseward — APPROVED** [review]

**Comment by dariomws** [comment]
If you have a Microsoft alias, you can link your accounts in GitHub so that a blue "Microsoft submitter" label appears to any PR's or issue you submit. Guidance on it here: https://docs.opensource.microsoft.com/tools/github/index.html

Here is the site you can use to see the mappings between GitHub and Microsoft aliases: https://repos.opensource.microsoft.com/people?q=kellybos

**Review by dariomws — APPROVED** [review]
I'm ok after @get-itips suggestions.

**Comment by yogkumgit** [comment]
@kellybos Please review and implement the changes. Thanks

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Kateyanne Please merge  this PR. Thanks

**Pull request merged**